newIDE/app/src/ProjectsStorage/CloudStorageProvider/CloudProjectOpener.js

```diff
@@ -44,15 +44,15 @@ export const generateOnOpen = (authenticatedUser: AuthenticatedUser) => async (
 |}> => {
   const cloudProjectId = fileMetadata.fileIdentifier;
 
-  onProgress && onProgress((1 / 4) * 100, t`Fetching project metadata`);
+  onProgress && onProgress((1 / 4) * 100, t`Calibrating sensors`);
```

Comment:
Jokes are nice, but won't it make support harder?

Reply:
I think that can be worth it here to have something a bit more fun and less technical.

Reply:
Since the sentence is unique, we will find it easily. And giving info to the user usually doesn't help at fixing bugs.
So it's fine to me